test/functional/commands/touch-e2e-specs.js

```diff
@@ -411,134 +411,134 @@ describe('touch actions -', function () {
 
   describe('mjsonwp touch actions', function () {
     describe('multi touch actions', function () {
```

Comment:
why not just add skips and a todo so we know what needs to happen here?

Reply:
these tests use wd imports. Leving them uncommented would require to rewrite the source to use wdio instead.